src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -359,6 +380,12 @@ public void setMaxCitizens(final int newMaxCitizens)
         this.maxCitizens = newMaxCitizens;
     }
 
+    @Override
+    public void setPotentialMaxCitizens(final int newPotentialMax)
+    {
+        this.potentialMaxCitizens = newPotentialMax;
```

Comment:
Curious question, why do we have set methods for this and normal maxCitizens above when these values are always set or reset by the calculate method in the same class? Or why public in the least?

Reply:
Answer: We have to rework this handling of the sleeping slots, but not in this PR, in general this has to be refreshed and centralized